catalog/spark/spark.bom

```diff
@@ -91,6 +96,13 @@ brooklyn.catalog:
         minDisk: 100gb
         osFamily: centos
         osVersionRegex: 7
+        customizer:
+          $brooklyn:object:
+            type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+            object.fields:
+              tcpPortRanges:
+                - "4040-4050"
+              enabled: $brooklyn:config("spark.sharedsecuritygroup.create")
```

Comment:
I think 4040-4050 is sufficient because the rest of the ports should be opened in the *.port config https://github.com/brooklyncentral/brooklyn-spark/pull/19/files#diff-e3b8a2c17f156116494d47c3f7ca3c4dR86